lib/Parser.v3.js

```diff
@@ -88,6 +88,9 @@ export class ParserV3 extends ParserBase {
     if (data && data.content) {
       const mimeTypes = Object.keys(data.content)
       mimeTypes.forEach(mimeType => this.config.contentTypes.add(mimeType));
+      if (mimeTypes.length === 0) {
```

Comment:
The fix can already be before the `forEach` as an empty foreEach does not do much ;-)

Reply:
With this fix we also need a test to stay on 100% coverage. (which somehow is not picked up by CI  :-()

If you prefer I will let you do the honours but I can easily fix both items myself ;-)
Just let me know.

Hans

Reply:
Rusty,

fix is in CI as we speak, thanks for highlighting the issue ;-)

Reliably generating typescript types from JSON schema is a bit of a challenge as discussed in #201 
For now I think the ecosystem is not there yet.
However if it improves over time I am more than happy to reconsider.

Kind regards,
Hans

Reply:
Fix has been merged. The current v2.6.0 on NPM does not have the issue.

Thanks again,
Hans